src/util/declaration-writer.ts

```diff
@@ -130,5 +130,14 @@ export async function getNamespacePrelude(namespace: string) {
   // Replaces the namespace with the provided namespace
   prelude = prelude.replace(/\$\$NAMESPACE\$\$/g, namespace);
 
+  if (isNewClient) {
+    const imports = await fs.readFile(IMPORTS_PATH, 'utf-8');
```

Comment:
Instead of having to read a different file just for a single line import, please remove that file and simply use a literal string as I did in line 36.